### PR TITLE
LRSD-6855 [CP] Add Callback Form suggestions

### PR DESCRIPTION
**Pull request by JoMaAlves**

### CODE REVIEW

**Comment by JoMaAlves** [comment]
ci:test:sf

File: workspaces/liferay-customer-workspace/client-extensions/liferay-customer-site-initializer-code/site-initializer/fragments/group/liferay-customer/customer-portal/fragments/callback-form-organization-select/index.js [hunk 1]
```diff
@@ -269,6 +269,7 @@ function filterRemoteOptions(query, abortController) {
 	}
 
 	const url = new URL(input.attributes.relationshipURL);
+	url.searchParams.set('sort', 'name:asc');
```
Comment (ryanschuhler) [inline workspaces/liferay-customer-workspace/client-extensions/liferay-customer-site-initializer-code/site-initializer/fragments/group/liferay-customer/customer-portal/fragments/callback-form-organization-select/index.js:272]: @JoMaAlves can we move this line down 1 line to have it sorted alphabetically? If it needs to come first that is fine, we would just need to add another line break after the line

Comment (JoMaAlves) [reply workspaces/liferay-customer-workspace/client-extensions/liferay-customer-site-initializer-code/site-initializer/fragments/group/liferay-customer/customer-portal/fragments/callback-form-organization-select/index.js:272]: Sure, no problem!

**Comment by ryanschuhler** [comment]
ci:forward

**Pull request closed without merging**